interns-react-weekly-assignments/src/components/Home/Ritika/Week3/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import CounterApp from './CounterComponent/Counter'
+import AutoFocus from './AutoFocus/AutoFocus';
+import FriendStatus from './FriendStatus/FriendStatus';
+
+const Index = () => {
```

Comment:
Add proper name for the function instead of index. JSX function names should imply what actually the component is